### PR TITLE
docs(login examples): Path Fixes for  login and firebase login examples

### PR DESCRIPTION
**Pull request by limonkufu**
## Status

READY

## Breaking Changes

NO

## Description

I've started flutter very recently and was just reading through Bloc which is an excellent project! 

The PR includes some simple updates to paths in flutter login examples for the packages and fixes key points which were wrongly suggested Cubic and Bloc usage in the login and firebase login tutorials. Finally, I've replaced `flutter packages get` with `flutter pub get` as it's more recent and widely used to eliminate confusion in newcomers like me.

## Type of Change

- [ ] ✨ New feature (non-breaking change which adds functionality)
- [ ] 🛠️ Bug fix (non-breaking change which fixes an issue)
- [ ] ❌ Breaking change (fix or feature that would cause existing functionality to change)
- [ ] 🧹 Code refactor
- [ ] ✅ Build configuration change
- [x] 📝 Documentation
- [ ] 🗑️ Chore

### CODE REVIEW

File: docs/flutterlogintutorial.md [hunk 1]
```diff
@@ -123,7 +117,7 @@ Let's start by updating the generated `pubspec.yaml` at the root of our project:
 We can install the dependencies by running:
 
 ```sh
-flutter packages get
+flutter pub get
```
Comment (felangel) [inline docs/flutterlogintutorial.md:120]: flutter packages get should be fine as well.

File: docs/flutterfirebaselogintutorial.md [hunk 2]
```diff
@@ -35,14 +35,14 @@ The `AuthenticationRepository` will be responsible for abstracting the internal
...
-We'll start by creating `packages/authentication_repository` and a `pubspec.yaml`.
+We'll start by creating `packages/authentication_repository` in the root project folder and a `pubspec.yaml` under it.
 
 [pubspec.yaml](https://raw.githubusercontent.com/felangel/bloc/master/examples/flutter_firebase_login/packages/authentication_repository/pubspec.yaml ':include')
 
 Next, we can install the dependencies by running:
 
 ```sh
-flutter packages get
+flutter pub get
```
Comment (felangel) [inline docs/flutterfirebaselogintutorial.md:45]: flutter packages get should be fine as well.

File: docs/flutterfirebaselogintutorial.md [hunk 3]
```diff
@@ -35,14 +35,14 @@ The `AuthenticationRepository` will be responsible for abstracting the internal
 
 ### Setup
 
-We'll start by creating `packages/authentication_repository` and a `pubspec.yaml`.
+We'll start by creating `packages/authentication_repository` in the root project folder and a `pubspec.yaml` under it.
```
Comment (felangel) [inline docs/flutterfirebaselogintutorial.md:38]:
```suggestion
We'll start by creating `packages/authentication_repository` and a `pubspec.yaml` at the root of the project.
```

File: docs/flutterlogintutorial.md [hunk 4]
```diff
@@ -25,17 +25,11 @@ We'll start off by creating a brand new Flutter project
...
-```sh
-flutter packages get
-```
-
 ## Authentication Repository
 
 The first thing we're going to do is create an `authentication_repository` package which will be responsible for managing the authentication domain.
 
-We'll start by create a `packages/authentication_repository` directory at the root of the project which will contain all internal packages.
+We'll start by creating `packages/authentication_repository` directory at the root of the project which will contain all internal packages.
```
Comment (felangel) [inline docs/flutterlogintutorial.md:32]:
```suggestion
We'll start by creating a `packages/authentication_repository` directory at the root of the project which will contain all internal packages.
```

File: docs/flutterfirebaselogintutorial.md [hunk 5]
```diff
@@ -35,14 +35,14 @@ The `AuthenticationRepository` will be responsible for abstracting the internal
...
-We'll start by creating `packages/authentication_repository` and a `pubspec.yaml`.
+We'll start by creating `packages/authentication_repository` and a `pubspec.yaml` at the root of the project.
 
 [pubspec.yaml](https://raw.githubusercontent.com/felangel/bloc/master/examples/flutter_firebase_login/packages/authentication_repository/pubspec.yaml ':include')
 
 Next, we can install the dependencies by running:
 
 ```sh
-flutter packages get
+flutter pub get
```
Comment (felangel) [inline docs/flutterfirebaselogintutorial.md:45]:
```suggestion
flutter packages get
```

File: docs/flutterlogintutorial.md (hunk 1 above)
Comment (felangel) [inline docs/flutterlogintutorial.md:120]:
```suggestion
flutter packages get
```

File: docs/flutterfirebaselogintutorial.md [hunk 6]
```diff
@@ -10,7 +10,7 @@
 
 - [BlocProvider](/flutterbloccoreconcepts?id=blocprovider), a Flutter widget which provides a bloc to its children.
 - [BlocBuilder](/flutterbloccoreconcepts?id=blocbuilder), a Flutter widget that handles building the widget in response to new states.
-- Using Bloc instead of Cubit. [What's the difference?](/coreconcepts?id=cubit-vs-bloc)
+- Using Cubit instead of Bloc. [What's the difference?](/coreconcepts?id=cubit-vs-bloc)
```
Comment (felangel) [inline docs/flutterfirebaselogintutorial.md:13]:
```suggestion
- Using Cubit and Bloc. [What's the difference?](/coreconcepts?id=cubit-vs-bloc)
```

File: docs/flutterlogintutorial.md [hunk 7]
```diff
@@ -10,7 +10,7 @@
 
 - [BlocProvider](/flutterbloccoreconcepts?id=blocprovider), Flutter widget which provides a bloc to its children.
 - [BlocBuilder](/flutterbloccoreconcepts?id=blocbuilder), Flutter widget that handles building the widget in response to new states.
-- Using Cubit instead of Bloc. [What's the difference?](/coreconcepts?id=cubit-vs-bloc)
+- Using Bloc instead of Cubit. [What's the difference?](/coreconcepts?id=cubit-vs-bloc)
```
Comment (felangel) [inline docs/flutterlogintutorial.md:13]:
```suggestion
- Using Cubit and Bloc. [What's the difference?](/coreconcepts?id=cubit-vs-bloc)
```

**Review by felangel — DISMISSED** [review]
LGTM ✅ 
Thanks so much for the contribution! 💯

File: docs/flutterlogintutorial.md [hunk 8]
```diff
@@ -25,17 +25,11 @@ We'll start off by creating a brand new Flutter project
 flutter create flutter_login
 ```
```
Comment (felangel) [inline docs/flutterlogintutorial.md:26]:
```suggestion
```

Next, we can install all of our dependencies

```sh
flutter packages get
```
```

File: docs/flutterlogintutorial.md [hunk 9]
```diff
@@ -25,17 +25,11 @@ We'll start off by creating a brand new Flutter project
...
 
-Next, we can install all of our dependencies
-
-```sh
-flutter packages get
-```
-
 ## Authentication Repository
```
Comment (felangel) [inline docs/flutterlogintutorial.md:28]:
```suggestion
Next, we can install all of our dependencies

```sh
flutter packages get
```

## Authentication Repository
```

File: docs/flutterlogintutorial.md [hunk 10]
```diff
@@ -29,13 +29,10 @@ Next, we can install all of our dependencies
 
 ```sh
 flutter packages get
```
Comment (felangel) [inline docs/flutterlogintutorial.md:31]:
```suggestion
flutter packages get
```

## Authentication Repository
```

**Review by felangel — APPROVED** [review]

**Comment by limonkufu** [comment]
Thanks for merging!

**Pull request merged**